### ISSUE TITLE
MySQL: Bad DS types for WSREP/Galera variables

### ISSUE THREAD

**Issue by SineSwiper** [collectd/collectd#4137]
* Version of collectd: latest in repo ([9369ed7](https://github.com/collectd/collectd/commit/9369ed71450430266ebd7ca410022c2b1b548fb4))
* Operating system / distribution: Any
* Kernel version (if applicable): Any

The [types for some of the WSREP values](https://github.com/collectd/collectd/blob/main/src/mysql.c#L660C1-L685C65) are wrong.  These keys and what they report are [documented here](https://galeracluster.com/library/documentation/galera-status-variables.html).  These types should be changed to `DS_TYPE_GAUGE`:

```
wsrep_apply_oooe
wsrep_apply_oool
wsrep_commit_oooe
wsrep_commit_oool
wsrep_flow_control_paused
```

The `wsrep_flow_control_paused` variable is especially important because it shows when the whole Galera cluster paused operations to sync data with its other nodes.  This value would be at 1 if 100% of operations were paused.

The `wsrep_flow_control_paused_ns` value would be the one that would act like a counter.

**Comment by eero-t** [comment]
Could you provide PR fixing that, which you've tested?

I have no idea whether any of the mysql plugin author(s) are still active, and even if they are, they might not have test setup for testing anything related to Galera(?), so it's better that fix is provided by somebody who can actually verify that the results are as expected...

**Comment by octo** [comment]
Hi @SineSwiper, thank for reporting this. Based on the documentation, I think you're correct. There is a fix in #4184. Unfortunately I don't have the means of testing it. Could you take a look? Thanks!
